src/core/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ find_package(nlohmann_json 3.2.0 REQUIRED)
 add_library(core-lib core.cpp)
 
 target_sources(core-lib PRIVATE core.cpp)
+target_sources(core-lib PRIVATE authenticator.cpp)
```

Comment:
Why don't add this source to the initial sources list in line 3?
Also why we are adding source which has already been added?